etn-ccis/blui-react-native-workflows
# Feature/blui 5159 registration workflow context

Fixes 5159



#### Changes proposed in this Pull Request:
- Demo screens will be removed in future
- Added registrationWorkflowContext
- Added demo screens
- Update Docs



#### Screenshots / Screen Recording (if applicable)

- 



Uploading With_CustomScreen.mp4…



Uploading Default_Workflow.mp4…



#### To Test:

-Select `RegistrationProviderExample` from the drawer menu and observe all data persists on the back and next button



#### Any specific feedback you are looking for?


- **This PR is only to showcase how registration workflow context persists data. Please do not focus on individual screen's UI or functionality.**
